var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -330,61 +402,90 @@ def filter_compilers(self):
             cc_compiler = self.spec["mpi"].mpicc
             cxx_compiler = self.spec["mpi"].mpicxx
 
-        libtool_makefile = join_path(self.prefix, "share/nrn/libtool")
-        nrniv_makefile = join_path(
-            self.prefix, self.archdir, "./bin/nrniv_makefile"
-        )
-        nrnmech_makefile = join_path(
-            self.prefix, self.archdir, "./bin/nrnmech_makefile"
-        )
+        libtool_makefile = join_path(self.prefix,
+                                     self.basedir,
+                                     "../share/nrn/libtool")
```

Comment:
The old way seems more Pythonic to me here… what would Black do?

Reply:
Done